XML/TestCases/FunctionalTests-SRIOV.xml

```diff
@@ -122,6 +122,35 @@
         <Tags>sriov,network</Tags>
         <Priority>0</Priority>
     </test>
+    <test>
+        <testName>SRIOV-INTERRUPTS-CHANGE</testName>
+        <setupScript>.\TestScripts\Windows\SETUP-Change-CPU.ps1,.\Testscripts\Windows\SETUP-SR-IOV-Configure.ps1</setupScript>
```

Comment:
This only can be apply in hyperv, this case belongs azure platform

Reply:
We knew this and would like to scale out to Azure.

Reply:
Actually I forgot to put the Hyper-V tag. I will put test results for Hyper-V too